apps/ui/src/components/TextList.tsx

```diff
@@ -21,7 +21,7 @@ import { List, theme } from 'antd';
 const { useToken } = theme;
 
 type TextProps = {
-	data: string[];
+	data: string[] | JSX.Element[];
```

Comment:
Do you just want a list of renderable nodes of type `React.ReactNode[]`?

Or should we allow a mix of JSX and strings: `(string | JSX.Element)[]`?